neutron-org/neutron
# feat: remove endblocker events

This PR:

1. Removes queries processing from the EndBlocker completely;
2. Adds events emitting to the Update, Create, Remove actions (queries);
3. Adds pagination and `connection_id`, `owner` filters to `GetRegisteredQueries()`;
4. Changes events type from "message" to "neutron";
5. Updates the `openapi.yml` swagger specs.

It is related to the following PRs:

1. https://github.com/neutron-org/neutron-contracts/pull/27
2. https://github.com/neutron-org/neutron-query-relayer/pull/35
3. https://github.com/neutron-org/neutron-integration-tests/pull/10

**HOW TO TEST THIS CODE?**

See the corresponding section in https://github.com/neutron-org/neutron-query-relayer/pull/35.

> NOTE: most of the huge diff is the generated swagger specs `openapi.yml`
